catalog/ui/catalog-ui-search/src/main/webapp/component/query-basic/query-basic.view.js

```diff
@@ -352,15 +356,56 @@ define([
                 })
             }));
         },
+        setupTimeRelative: function(){
+            var currentLast = '';
+            var currentUnit = '';
+            if (this.filter.anyDate) {
+                this.filter.anyDate.forEach(function(subfilter) {
+                    if (subfilter.type === '>=') {
+                        var extractedDuration = subfilter.value.match(/[1-9]+./g)[0];
+                        currentLast = extractedDuration.match(/[1-9]+/)[0];
+                        switch (extractedDuration.match(/[a-zA-Z]/)[0]) {
+                            case 'M':
+                                currentUnit = "Minutes";
+                                break;
+                            case 'H':
+                                currentUnit = "Hours";
+                                break;
+                            case 'D':
+                                currentUnit = "Days";
+                        }
+                    }
+                });
+            }
+            this.basicTimeRelativeValue.show(new PropertyView({
+                model: new Property({
+                    value: [currentLast || 0],
+                    id: 'Last',
+                    placeholder: 'Limit searches to between the present and this time.',
+                    type: 'INTEGER'
+                })
+            }));
+            this.basicTimeRelativeUnit.show(new PropertyView({
+                model: new Property({
+                    value: [currentUnit || 'Minutes'],
+                    enum: ['Minutes', 'Hours', 'Days'],
+                    id: 'Unit',
+                    placeholder: 'Limit searches to between the present and this time.',
+                    type: 'ENUM'
```

Comment:
❓ We have other drop downs that have a type of their selected value.  Why can this not be a type of String instead of adding this enum type?